### DISCUSSION TITLE
Assisting in training the recipe scraper to accept Danish units

### DISCUSSION THREAD

**Discussion by EsbenAAUEnergy** [mealie-recipes/mealie#2039] [Q&A]
First of all hi to you all!
I have stumpled upon Mealie, and wow am I impressed. I have tried other recipe apps, but I really enjoy using Mealie.
Since I am from Denmark, the recipe urls i scrape contain different units than what the scraper seems to accept. Since this seems to be an opensource partly community driven app, I would really enjoy to be able to help out. Is there any way for me to assist in this? I have prior knowledge in the most basic programming languages, and I am a fast learner. Can any of you point me in a direction, or tell me wether it is even possible to update or train the scraper in other than the standard units?

Nice evening to you all.

- Esben

**Comment by michael-genson** [comment]
Currently your best bet is the Brute Force parser, it seems to work well with other languages. Alternatively you can try out the new OpenAI parser: https://docs.mealie.io/documentation/getting-started/installation/backend-config/#openai
